.github/workflows/ci.yml

```diff
@@ -4,9 +4,10 @@ on:
   push:
     branches:
       - main
+      - organize-evals
```

Comment:
this will only run when you push to `main` or `organize-evals`. We want this to run on a push to any branch ([see Slack thread](https://browserbase.slack.com/archives/C074FSMUDMF/p1733425862438849))